src/main/java/de/dhbw/mh/slang/craft/parser/CraftedSlangParser.java

```diff
@@ -232,32 +234,56 @@ AstNode relations5( AstNode previous ){
 	
 	@Override
 	public AstNode additiveExpression( ){
-		// TODO Auto-generated method stub
-		return super.additiveExpression( );
+		if( Selector.SUMMAND.contains(LEXER.lookahead().TYPE) ){
+			AstNode previous = multiplicativeExpression();
+			return summand( previous );
+		}
+		throw parsingException( Selector.SUMMAND );
 	}
 	
 	@Override
 	public AstNode summand( AstNode previous ){
-		// TODO Auto-generated method stub
-		return super.summand( previous );
+		Token token = LEXER.lookahead();
+		if( Selector.SUMMAND3.contains(token.TYPE) ){
+			return summand3( previous );
```

Comment:
I would prefer the function calls to be sorted according to their ordinal number, but that's just cosmetics 🙂